lms/djangoapps/courseware/toggles.py

```diff
@@ -85,6 +85,22 @@
     WAFFLE_FLAG_NAMESPACE, 'mfe_progress_milestones_first_section_celebration', __name__
 )
 
+# .. toggle_name: courseware.mfe_progress_milestones_streak_celebration
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to display a celebration modal when learner completes a configurable streak
+#   Supports staged rollout to students for a new micro-frontend-based implementation of the
+#   courseware page.
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2021-02-03
```

Comment:
Not sure if it actually matters, but might want to update this.

Reply:
Yeah, I have no idea how important it is, but I figured the close to reality, the better?